contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,130 @@
+
+        # this handles the message if its starts with @comment
+        if original_content.startswith('@comment'):
+
+            params = get_values_message(original_content)
+
+            status_code = send_to_github(params['user'], params['repo_owner'], params['repo'],
+                                         params['issue'], params['comment_body'])
+
+            if status_code == 201:
+                # sending info to github was successful!
+                reply_message = "You commented on issue number " + params['issue'] + " under " + \
+                                params['repo_owner'] + "'s repository " + params['repo'] + "!"
+
+                client.send_message(dict(
+                    type='private',
+                    to=original_sender,
+                    content=reply_message,
+                ))
+
+            else:
+                # sending info to github did not work
+                reply_message = ":( There was a problem commenting on issue number " + params['issue'] + " under " + \
+                                params['repo_owner'] + "'s repository " + params['repo'] + \
+                                ". Did you enter the information in the correct format?"
+
+                client.send_message(dict(
+                    type='private',
+                    to=original_sender,
+                    content=reply_message,
+                ))
+
+        # handle if message starts with @github-comment
+        if original_content.startswith('@github-comment'):
+
+            params = get_values_message(original_content)
+
+            status_code = send_to_github(params['user'], params['repo_owner'], params['repo'],
+                                         params['issue'], params['comment_body'])
+
+            if status_code == 201:
+                # sending info to github was successful!
+
+                reply_message = "You commented on issue number " + params['issue'] + " under " + \
+                                params['repo_owner'] + "'s repository " + params['repo'] + "!"
+
+                client.send_message(dict(
+                         type='private',
+                         to=original_sender,
+                         content=reply_message,
+                     ))
+
+            else:
+                # sending info to github did not work
+                reply_message = ":( There was a problem commenting on issue number " + params['issue'] + " under " + \
+                                params['repo_owner'] + "'s repository " + params['repo'] + "."
+
+                client.send_message(dict(
+                         type='private',
+                         to=original_sender,
+                         content=reply_message,
+                     ))
+
+handler_class = GitHubHandler
+
+
+def send_to_github(user, repo_owner, repo, issue, comment_body):
+    session = requests.session()
+
+    path_to_token = '/home/andrew/github_token.txt'
+
```

Comment:
I'd think the path to token is not needed here - you can ask the user to always put it in a certain folder (i.e. have a default folder path) and add additional running parameters for the path to the token, should they want to put it somewhere else.